tools/scripts/ManifestBuilder.rb

```diff
@@ -283,7 +283,10 @@ def build_animation_metadata(animation_objects, previous_metadata)
 
     # Parallelize metadata construction because some objects will require an
     # extra S3 request to get version IDs or image dimensions.
-    Parallel.map(animation_objects.keys, finish: lambda do |name, _, result|
+    #
+    # TODO: remove 'in_threads: 0' once the i18n-dev server RAM has been increased over 8 GB
+    # https://codedotorg.atlassian.net/browse/FND-1460
+    Parallel.map(animation_objects.keys, in_threads: 0, finish: lambda do |name, _, result|
```

Comment:
What was the default number of threads? Can we use 2 threads instead of disable parallelization?

Reply:
Looking at the Parallel gem, it seems the default is 2 threads. There are 2 vCPUs on the host so there would be 2 x 2 = 4 Threads total before. With this change it will be 2 x 1 = 2 Threads total. It's a pretty small reduction in memory usage, I think we are very borderline now. We will need to work on upgrading the host ASAP.